pyproject.toml

```diff
@@ -32,6 +32,23 @@ module = [
 ]
 ignore_missing_imports = true
 
+[tool.pyright]
+include = [
+    "bin/tmt",
+    "setup.py"
+]
+ignore = [
+    "docs/**",
+    "examples/**",
+    "tests/**",
+    "tmt/**"
+]
+
+pythonVersion = "3.7"
```

Comment:
Not really related to this PR, but it might be worth thinking about using '3.6' for mypy and pyright, even though ruff is only 3.7+. It seems there are no clashes, even with more ruff checks enabled...